tests/wpt/mozilla/tests/css/get-computed-style-for-url.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Computed styles for URLs use the resolved URL and specified styles use the original URL</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="tests"></div>
+<script>
+  var origin = window.location.origin;
+  var container = document.getElementById("tests");
+
+  // Some shorthands like background and border-image compute to other properties
+  var computedPropertyName = {
+    background: "backgroundImage",
+    borderImage: "borderImageSource",
+    listStyle: "listStyleImage",
+  };
+
+  function makeComputedUrlStyle(path) {
+    return `url("${origin}/${path}")`;
+  }
+  function makeSpecifiedUrlStyle(path) {
+    return `url("${path}")`;
+  }
+  function testUrlsForProperty(property, urlValue, extraShorthand) {
+    test(function() {
+      var extra = extraShorthand ? ` ${extraShorthand}` : "";
```

Comment:
(But setting the longhand property directly would allow you to avoid this complexity).